test/ad.jl/adr.jl

```diff
@@ -23,8 +23,8 @@ mvn = collect(Iterators.filter(vn -> vn.sym == :m, keys(vi)))[1]
 _s = getval(vi, svn)[1]
 _m = getval(vi, mvn)[1]
 
-x = map(_->Float64(_), vi[nothing])
-∇E = gradient_reverse(x, vi, ad_test_f)
+x = map(_x->Float64(_x), vi[nothing])
+∇E = gradient_reverse(x, vi, ad_test_f)[2]
```

Comment:
I would prefer if `gradient_reverse` would still behave as before. Is there a reason to change it's behavior?

Reply:
This was not doing the expected thing, it was giving an error. I just fixed the error.

Reply:
Wait this may be a bug. I will look into it.